app.py

```diff
@@ -20,6 +20,13 @@
 app = Flask(__name__)
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 
+# Enable to print SQL statements using get_debug_queries.
```

Comment:
This is useful for tracing actual SQL statements generated by SQLALCHEMY library.